lib/screens/UsersProfileScreen.dart

```diff
@@ -0,0 +1,1576 @@
+import 'package:flutter/material.dart';
+import 'dart:convert';
+import 'package:smart_chef/utils/APIutils.dart';
+import 'package:smart_chef/utils/colors.dart';
+import 'package:smart_chef/utils/userAPI.dart';
+import 'package:smart_chef/utils/authAPI.dart';
+import 'package:http/http.dart' as http;
+
+class UserProfileScreen extends StatefulWidget {
+  @override
+  _UserProfileState createState() => _UserProfileState();
+}
+
+class _UserProfileState extends State<UserProfileScreen> {
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return UserProfilePage();
+  }
+}
+
+class UserProfilePage extends StatefulWidget {
+  @override
+  _UserProfilePageState createState() => _UserProfilePageState();
+}
+
+class _UserProfilePageState extends State<UserProfilePage> {
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  String errorMessage = '';
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text(
+          'SmartChef',
+          style: TextStyle(fontSize: 24, color: mainScheme),
+        ),
+        centerTitle: true,
+        backgroundColor: Colors.white,
+        leading: IconButton(
+          onPressed: () async{
+            //TODO Finish registration verification once API is done
```

Comment:
I think I've said it elsewhere, but pls fix all this to include an issue number and have the correct format `// TODO(<num here>): descrip`